index.html

```diff
@@ -1,9 +1,90 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Neverending Pun Machine</title>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="./style.css" />
+    </head>
+    <body>
+        <header>
+            <div>
+              <div>
+                <img class="blackHeart" src="images/black-heart.png" alt="black heart" />
+                <span> Nifty Penguin Magic </span>
+              </div>
+              <nav>
+                <ul>
+                  <li><a href="#"> npm Enterprise </a></li>
+                  <li><a href="#"> Products </a></li>
+                  <li><a href="#"> Solutions </a></li>
+                  <li><a href="#"> Resources </a></li>
+                  <li><a href="#"> Docs </a></li>
+                  <li><a href="#"> Support </a></li>
+                </ul>
+              </nav>
+            </div>
+            <div class="underbar">
+                <img class="npmLogo" src="images/npm-logo.png" alt="npm logo" />
+                <form method="post" action=# class="search-form">
+                    <img class="lupa" src="images/magnifying-glass.png" alt="search image" />
+                    <input type="text" placeholder=" Search Packages" class="search-form">
+                </form>
+                <button type="submit" class="search-btn">Search</button>
+                <div>
```

Comment:
When loading the page in Chrome, these buttons are not well aligned. Could setting a "display" value improve the alignment?

Reply:
<img width="497" alt="Screenshot 2020-10-08 at 17 31 00" src="https://user-images.githubusercontent.com/71708926/95480356-18bc5800-098c-11eb-91ad-41a48035ba7b.png">
Confused, as they look correct on my chrome.
Which display value would make them fixed, in any case?